<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RefereeWindow</class>
 <widget class="QMainWindow" name="RefereeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>1500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RefereeWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Dartboard" name="DartboardView">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>20</y>
      <width>1300</width>
      <height>1300</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../rsrc/resources.qrc">:/img/Dartboard.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Koordinaten">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>X: = 0 , Y:= 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="undoLetzterWurf">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>510</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>UNDO letzter Wurf</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>60</y>
      <width>231</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelWurf1">
         <property name="text">
          <string>Wurf 1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="wurf1">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelWurf2">
         <property name="text">
          <string>Wurf 2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="wurf2">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelWurf3">
         <property name="text">
          <string>Wurf 3:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="wurf3">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="naechsterSpieler">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>430</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>NÃ¤chster Spieler</string>
    </property>
   </widget>
   <widget class="QLabel" name="Multiplikator">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>370</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Multiplikator</string>
    </property>
   </widget>
   <widget class="QLabel" name="Winkel">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>600</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Winkel</string>
    </property>
   </widget>
   <widget class="QLabel" name="DeltaX">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>670</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Winkel</string>
    </property>
   </widget>
   <widget class="QLabel" name="DeltaY">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>730</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Winkel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSchiedsrichter_Fenster">
    <property name="title">
     <string>Schiedsrichter Fenster</string>
    </property>
   </widget>
   <addaction name="menuSchiedsrichter_Fenster"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Dartboard</class>
   <extends>QLabel</extends>
   <header location="global">View/dartboard.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../rsrc/resources.qrc"/>
 </resources>
 <connections/>
</ui>
